pkg/operator/ceph/csi/spec.go

```diff
@@ -136,7 +136,7 @@ var (
 	DefaultProvisionerImage = "registry.k8s.io/sig-storage/csi-provisioner:v5.0.1"
 	DefaultAttacherImage    = "registry.k8s.io/sig-storage/csi-attacher:v4.6.1"
 	DefaultSnapshotterImage = "registry.k8s.io/sig-storage/csi-snapshotter:v8.2.0"
-	DefaultResizerImage     = "registry.k8s.io/sig-storage/csi-resizer:v1.11.1"
```

Comment:
can you please update the other images here as well?

Reply:
ah, sure!